packages/components/src/components/telekom/telekom-footer-extended-navigation/telekom-footer-extended-navigation.css

```diff
@@ -43,10 +43,12 @@
   font-size: var(--telekom-typography-font-size-body);
   color: var(--link-color);
   margin-top: var(--telekom-spacing-unit-x5);
+  border: none;
+  line-height: 22px;
```

Comment:
this I think we can remove, because the default line height (22.4) is already working fine withtout that ghosty border 🙂

Reply:
done!